opal/mca/shmem/posix/shmem_posix.h

```diff
@@ -30,10 +30,10 @@
 /* max number of attempts to find an available shm_open file name. see
  * comments below for more details.
  */
-#define OPAL_SHMEM_POSIX_MAX_ATTEMPTS 128
+#define OPAL_SHMEM_POSIX_MAX_ATTEMPTS 256
```

Comment:
With the change in the name to avoid collisions why do we need to also increase the number of attempts ?

Reply:
Yes, no need to do this now. The increase was the first thing I did, before deciding to change the naming scheme. Reverted that change.